niworkflows/interfaces/nibabel.py

```diff
@@ -89,3 +89,67 @@ def _run_interface(self, runtime):
         maskimg.to_filename(self._results['out_mask'])
 
         return runtime
+
+class _Save4Dto3DInputSpec(BaseInterfaceInputSpec):
+    in_file = File(exists=True, mandatory=True, desc='input 4d image')
+
+
+class _Save4Dto3DOutputSpec(TraitedSpec):
+    out_files = OutputMultiPath(File(exists=True),
+                                     desc='output list of 3d images')
+
+
+class Save4Dto3D(SimpleInterface):
```

Comment:
I would say either call it by the nibabel name:

```suggestion
class FourToThree(SimpleInterface):
```

Or something more standard:

```suggestion
class Split(SimpleInterface):
```

`Save` sounds like a datasink.